ModulePresentationsForCAP/gap/ModulePresentationFunctors.gi

```diff
@@ -158,3 +158,60 @@ InstallMethod( FunctorLessGeneratorsRight,
     return functor;
     
 end );
+
```

Comment:
Is there a reason this functor does not uses the `DualOnObject` and `DualOnMorphism` functions?